client/components/ui/rich-editor/CoverImage.tsx

```
@@ -0,0 +1,231 @@
+  if (!coverImage) {
+    return null
+  }
+
+  // Show cover image with controls
+  return (
+    <div
+      ref={containerRef}
+      className="group absolute top-0 mb-8 h-[300px] w-full overflow-hidden rounded-lg lg:h-[420px]"
+      onMouseEnter={() => !readOnly && setIsHovered(true)}
+      onMouseLeave={() => !readOnly && setIsHovered(false)}
+    >
```

Biome (2.1.2)

[error] Static Elements should not be interactive.

To add interactivity such as a mouse or key event listener to a static element, give the element an appropriate role value.

(lint/a11y/noStaticElementInteractions)